src/index.ts

```diff
@@ -0,0 +1,3 @@
+import server from "./server";
+
+server();
```

Comment:
Thinking about the default Port: maybe we should enforce the user to pass a port here - we have so many issues with `4200` in dcos-ui, enfocing the user to explicitly set the port could solve this 🤔  - especually when thinking about this mockserver as being started multiple times in parallel…

=> I'd change this to `server(Number.parseInt(process.env.PORT))` and remove `getPort` completely…

Reply:
That makes it harder to test, but we can make it explicit here. We can also change the getPort function to throw an error if there is no port given, that would be fine with me too. But let's fix this forward I would say

Reply:
why does this change make it harder to test? in tests we can import `server` from `./server` directly instead of using `index.ts` 🤔 

nvm, fix foward is ok 👍